src/tailwind/defaultConfig.ts

```diff
@@ -506,6 +507,8 @@ export default {
       modalLarge: "770px",
       checkbox: "24px",
       radioIndicator: "12px",
+      110: "110px",
+      200: "200px",
```

Comment:
if the values are used only for 1 component then use the CSS module

Reply:
done